src/debugAdapter/goDebug.ts

```diff
@@ -781,6 +776,11 @@ class GoDebugSession extends LoggingDebugSession {
 
 	protected disconnectRequest(response: DebugProtocol.DisconnectResponse, args: DebugProtocol.DisconnectArguments): void {
 		log('DisconnectRequest');
+		// For remote process, we have to issue a continue request
+		// before disconnecting.
```

Comment:
Is it enough that the continue request is issued?
Do we need to wait for the response before moving ahead?